LiskHQ/lisk-sdk
# Fix network config - Closes #3466

### What was the problem?
Network config was outdated with the new structure.

### How did I fix it?
Update the keys for the network configs and fix the update_config script.

### How to test it?
Create sample script to migrate
`LISK_NETWORK=mainnet node scripts/update_config.js config_sample.json 1.6.0 2.0.0`
Also, check the config keys

### Review checklist

* The PR resolves #3466 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
